data/core/units/wose/Wose_Shaman.cfg

```diff
@@ -18,7 +18,7 @@
     {AMLA_DEFAULT}
     cost=27
     usage=mixed fighter
-    description=_"These woses are able to command forest plants such as vines and creepers to hinder their enemies."
+    description=_"Some woses have spent many years specializing in the commandment of forest plants, and are often revered among their kin for their abilities. Their powers are known to be different from the magic used by elves or merfolk, though the details of their craft can only be speculated on by other races. Their existence is known primarily only from a handful of historical encounters, where they are said to be able to command nearby vines and creepers to hinder their enemies in combat, though a few question the validity of these encounters. Though woses are generally believed to not practice any magic of their own, some believe these shamans use some kind of weak magic unique to their people."
```

Comment:
"Years" seems short for a Wose. "Centuries" seems a little long, yet a better fit than either years or decades.